Migration.Toolkit.Core/Services/AttachmentMigrator.cs

```diff
@@ -57,6 +57,23 @@ IProtocol protocol
     public record MigrateAttachmentResult(bool Success, bool CanContinue, MediaFileInfo? MediaFileInfo = null,
         MediaLibraryInfo? MediaLibraryInfo = null);
 
+    public MigrateAttachmentResult TryMigrateAttachmentByPath(string documentPath, string additionalPath)
+    {
+        if(string.IsNullOrWhiteSpace(documentPath)) return new MigrateAttachmentResult(false, false);
+        documentPath = $"/{documentPath.Trim('/')}";
+
+        using var kx13Context = _kx13ContextFactory.CreateDbContext();
+        var attachment =
+            kx13Context.CmsAttachments
+                .Include(a => a.AttachmentDocument)
+                .ThenInclude(d => d.DocumentNode)
+                .FirstOrDefault(a => documentPath.StartsWith(a.AttachmentDocument.DocumentNode.NodeAliasPath));
```

Comment:
Is it safe to find attachment by node alias path? I think it is possible to have one attachment with NodeAliasPath = /Directory/SubDirectory/SubSubDirectory and another with NodeAliasPath = /Directory/SubDirectory and if documentPath will have value eg. /Directory or /Directory/SubDirectory I will receive the first attachment, right?